Showcase-iOS/Login/View/LoginNavigationController.swift

```diff
@@ -20,5 +20,6 @@ class LoginNavigationController: UINavigationController {
         var loginPresenter = DependencyContainer.container().resolve(LoginPresentable.self)
         loginPresenter?.loginViewer = loginViewController
         loginViewController?.loginPresenter = loginPresenter
+        loginViewController?.sharedApplication = SharedApplicationDelegateImplementation()
```

Comment:
Lets make use of the container. Though will refactor it later.

let dependencyContainer = DependencyContainer.container()
loginViewController?.loginPresenter = dependencyContainer.resolve(LoginPresentable.self)
loginViewController?.sharedApplication = dependencyContainer.resolve(SharedApplicationDelegate.self)